libfwupd/fwupd-release.c

```diff
@@ -545,36 +545,82 @@ fwupd_release_get_metadata_item (FwupdRelease *release, const gchar *key)
  * fwupd_release_get_uri:
  * @release: A #FwupdRelease
  *
- * Gets the update uri.
+ * Gets the default update URI.
  *
- * Returns: the update uri, or %NULL if unset
+ * Returns: the update URI, or %NULL if unset
  *
  * Since: 0.9.3
+ * Deprecated: 1.5.6: Use fwupd_release_get_locations() instead.
```

Comment:
you need to upgrade `src/fu-tool.c` too (And we probably need CI to be yelling about the fact you missed it)

Reply:
presumably `libfwupd/fwupd-client.c` should be too?

Reply:
> we probably need CI to be yelling about the fact you missed it

https://github.com/fwupd/fwupd/pull/2807

> presumably libfwupd/fwupd-client.c should be too?

All fixed, thanks.